_includes/navigation.html

```diff
@@ -26,6 +26,10 @@ <h3>Navigation</h3>
       <a class="nav-link {% if page.title == "HyperDrive" %}active{% endif %}"
         href="{{ site.baseurl }}/hyperdrive">HyperDrive</a>
     </li>
+    <li class="nav-item">
```

Comment:
This navigation panel and the links here should be used in the scope of the absaoss.github.io main documentation and then have more information about the respective project on that small one page with a link to the projects main documentation.

### Example:
_includes/navigation.html
```html
<li class="nav-item">
  <a class="nav-link {% if page.title == "k8gb" %}active{% endif %}"
  href="{{ site.baseurl }}/k8gb">k8gb</a>
</li>
```

new file **k8gb.md**
```
---
layout: default
title: k8gb
---

This is k8gb .... (abstract)
Here is the main documentation https://www.k8gb.io
something else ....
```

Alos, I think that clicking on the navigation menu to be redirected to another side might end up in confusion.

Reply:
@Zejnilovic Thanks a lot for the review. PR amended, please check

Reply:
@ytsarev thank you for your work. I also tried giving you a bit more privileges. So you should be able to merge this when you are happy with it.